<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicCalculator</class>
 <widget class="QDialog" name="BasicCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Операции</string>
   </property>
   <widget class="QRadioButton" name="rB_plus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_minus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_div">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_mult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_cos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_tan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_ctan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>ctan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_arcsin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>arcsin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_arccos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>arccos</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_arctan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>arctan</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rB_arcctan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>arcctan</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pB_get_result">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Решить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Округление до 4-х знаков после запятой</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_degree">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Угол в градусах!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_num2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Число 2:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-9999999999999999538762658202121142272.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999999999538762658202121142272.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_num1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Число 1:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-9999999999999999538762658202121142272.000000000000000</double>
    </property>
    <property name="maximum">
     <double>9999999999999999538762658202121142272.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
